src/main/java/com/laserfiche/api/client/model/OffsetDateTimeDeserializer.java

```diff
@@ -0,0 +1,32 @@
+package com.laserfiche.api.client.model;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import org.threeten.bp.OffsetDateTime;
+
+import java.io.IOException;
+
+public class OffsetDateTimeDeserializer extends StdDeserializer<OffsetDateTime> {
```

Comment:
Again, similar to `ApiException`, aa reminder that we could probably ensure this one the same as the one in repository client and let the repository client uses this class.